package/gluon-core/luasrc/usr/lib/lua/gluon/platform.lua

```diff
@@ -42,6 +42,10 @@ function M.is_outdoor_device()
 		'tplink,wbs210-v1',
 		'tplink,wbs210-v2',
 		'tplink,wbs510-v1',
+		'ubnt,nanobeam-ac-xc',
+		'ubnt,nanobeam-m5-xw',
```

Comment:
probably not necessary this is an old name for this device that OpenWrt does not use anymore, it is just an alias for the `ubnt,nanostation-loco-m-xw` image

https://github.com/openwrt/openwrt/blob/66672e702189fa2200db0cf8d27e4a6d320fe163/target/linux/ath79/image/generic-ubnt.mk#L142

Reply:
@herbetom oh! Indeed this was still an existing device in v2023.2.x/23.05 not an alias:
https://github.com/openwrt/openwrt/blob/openwrt-23.05/target/linux/ath79/image/generic-ubnt.mk#L93

https://github.com/openwrt/openwrt/commit/7b5aeef6dbfae724fa4bef4edee49be389fd4a5e
v2025.1.x/24.10 removed the extra image and merged the device with the `ubnt,nanostation-loco-m-xw`

so yeah, v2023.2.x requires the device and a line like this:
https://github.com/freifunk-gluon/gluon/blob/v2023.1.x/targets/ath79-generic#L604